admin/app/main/views/service_settings.py

```diff
@@ -827,9 +827,11 @@ def get_branding_as_value_and_label(email_branding):
 
 
 def get_branding_as_dict(email_branding):
+    # TODO: point this to our cdn when we make it
+
     return {
         branding['id']: {
-            'logo': 'https://{}/{}'.format(get_cdn_domain(), branding['logo']),
+            'logo': '//{}/static-logo/{}'.format(get_cdn_domain(), branding['logo']),
```

Comment:
I think we may need to be careful what we go "live" with and what our end solution for serving assets will be. e.g. if we end up setting up a CDN using AWS services (Route53, cloudfront) we may want to have a subdomain i.e. `static-logo.{base-url}/` rather than a relative path `{base-url}/static-logo/`

Reply:
i'll make a ticket.